crates/project/src/debugger/session.rs

```diff
@@ -996,6 +998,7 @@ impl Session {
     ) -> Self {
         Self {
             mode: Mode::Remote(RemoteConnection {
+                _adapter_name: SharedString::new(""), // todo(debugger) we need to pipe in the right values to deserialize the debugger pane layout
```

Comment:
I'm a bit unclear on how view synchronization will work WRT pane splits; should I see your layout when we're collaborating? What if I stop following you?